workflows/prognostic_run_diags/fv3net/diagnostics/prognostic_run/compute.py

```diff
@@ -399,16 +400,22 @@ def compute_histogram_bias(prognostic, verification, grid):
 
 
 def register_parser(subparsers):
-    parser = subparsers.add_parser("save", help="Compute the prognostic run diags.")
+    parser: ArgumentParser = subparsers.add_parser(
+        "save", help="Compute the prognostic run diags."
+    )
     parser.add_argument("url", help="Prognostic run output location.")
     parser.add_argument("output", help="Output path including filename.")
     parser.add_argument("--catalog", default=vcm.catalog.catalog_path)
-    parser.add_argument(
+    verification_group = parser.add_mutually_exclusive_group()
```

Comment:
Neat, I didn't know about this functionality.

Reply:
Could you add to the help for the verification options saying that they are mutually exclusive so this appears in the docs?

Reply:
Also, does group cause defaults get ignored if the other flag  is provided? i.e. if `--verification-url` is given, will it ignore the default of `--verification 40day_may2020`?

Reply:
Re. defaults. argparse does not do anything special apart from raising a helpful error if both flags are passed. The value of `args.verification` will still be the default value, so I implement the switching logic here: https://github.com/VulcanClimateModeling/fv3net/blob/96426df0c6ee83813f786c0305b3bfd29f02aaef/workflows/prognostic_run_diags/fv3net/diagnostics/prognostic_run/compute.py#L432-L433.

In terms of documentation, I added some explanation to the help string of `--verification-url` and the usage does indicate these options are mutually exclusive:
```
usage: prognostic_run_diags save [-h] [--catalog CATALOG]
                                 [--verification VERIFICATION | --verification-url VERIFICATION_URL]
                                 [--n-jobs N_JOBS]
                                 url output
```